libraries/Microsoft.Bot.Builder.LUIS/LuisModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Microsoft.Cognitive.LUIS;
+
+namespace Microsoft.Bot.Builder.LUIS
+{
+    public class LuisModel : ILuisModel
+    {
+        public LuisModel(string modelId, string subscriptionKey, Uri uriBase, LuisApiVersion apiVersion)
```

Comment:
Null checking + throw on all of these paremeters.

Reply:
Done